index.html

```diff
@@ -165,6 +165,21 @@ <h2>Skills</h2>
         </li>
       </ul>
       </section>
+      <section class="form">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form name="form1" action="https://formspree.io/f/myyovzdk" method="post">
+          <label >
+            <input type="text" name="name" id="name"  placeholder="Yeremias" maxlength="30" required>
+          </label> <br>
+          <label >
+           <input type="email" name="email-address" id="email" placeholder="natajayanj14@gmail.com" maxlength="30" required>
```

Comment:
- [ ] [optional] please it would be better to change the placeholder to something that make sense for name use `placeholder=Full name ` and for email  `placeholder="Email"`